vis_server/protocols/FileLoader.py

```diff
@@ -24,7 +24,17 @@ def list_variables(self, file_name):
         out = {}
         for vname in reader.variables:
             var = reader.variables[vname]
+
+            # Get a displayable name for the variable
+            if hasattr(var, 'long_name'):
+                name = var.long_name
+            elif hasattr(var, 'title'):
+                name = var.title
+            elif hasattr(var, 'id'):
```

Comment:
If the name is very long, using `long_name` at all will cause usability problems because it won't fit in the widget.  If the users are most familiar with the variable name provided in the file, is there a reason to making this change at all?